cocos/3d/lights/sphere-light-component.ts

```diff
@@ -42,7 +42,8 @@ export class SphereLight extends Light {
     @serializable
     protected _size = 0.15;
     @serializable
-    protected _luminance = 1700 / scene.nt2lm(0.15);
+    @formerlySerializedAs('_luminance')
+    protected _luminanceHDR = 1700 / scene.nt2lm(0.15);
```

Comment:
This way we can upgrade previous serialized data in `_luminance` to `_luminanceHDR`